pytorch_lightning/strategies/strategy.py

```diff
@@ -402,15 +402,18 @@ def lightning_module_state_dict(self) -> Dict[str, Union[Any, Tensor]]:
         model = self.lightning_module
         return model.state_dict()
 
-    def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: _PATH) -> None:
+    def save_checkpoint(
```

Comment:
A simple mock test for the base strategy would be appreciated

`mock_ckpt_io.assert_called_once_with(...)`